src/main/scala/firrtl/annotations/Annotation.scala

```diff
@@ -94,7 +94,7 @@ trait MultiTargetAnnotation extends Annotation {
   /** Contains a sequence of [[firrtl.annotations.Target Target]].
```

Comment:
Is this Scaladoc line (already) wrong?

Reply:
hahaha yes, didn't even notice. Will fix.